### PR TITLE
build: use npm rpc package and compile

### PR DESCRIPTION
**Pull request by samsiegart**
refs https://github.com/Agoric/agoric-sdk/issues/6870

### CODE REVIEW

File: wallet/config-overrides/index.js [hunk 1]
```diff
@@ -48,5 +48,14 @@ module.exports = function override(config, _env) {
   };
   const plug2 = new HtmlWebPackPlugin(opts);
   config.plugins.push(plug2);
+  config.module.rules = [
+    ...config.module.rules,
+    {
+      test: /\.m?js/,
+      resolve: {
+        fullySpecified: false,
```
Comment (turadg) [inline wallet/config-overrides/index.js:56]: I don't fully understand this but if it works it work

File: wallet/src/components/SmartWalletConnection.tsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import { makeFollower, makeLeader } from '@agoric/casting';
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (turadg) [inline wallet/src/components/SmartWalletConnection.tsx:2]: why is this line necessary? the dep is in package.json

File: wallet/src/index.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import '@endo/eventual-send/shim';
 
 // Ambient types. Needed only for dev but this does a runtime import.
 import '@agoric/ertp/exported';
-import '@agoric/notifier/exported';
+import '@agoric/notifier/exported.js';
```
Comment (turadg) [inline wallet/src/index.tsx:8]: consider updating the others too

File: wallet/src/util/WalletBackendAdapter.ts [hunk 4]
```diff
@@ -32,17 +37,60 @@ import { queryBankBalances } from './queryBankBalances';
...
+import type { ChainStorageWatcher } from '@agoric/rpc';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { AgoricChainStoragePathKind } from '@agoric/rpc';
 
 const newId = kind => `${kind}${Math.random()}`;
 const POLL_INTERVAL_MS = 6000;
 
 export type BackendSchema = {
-  actions: object;
+  actions: FarRef<{
```
Comment (turadg) [inline wallet/src/util/WalletBackendAdapter.ts:48]: nice

File: wallet/src/index.tsx (hunk 3 above)
Comment (samsiegart) [reply wallet/src/index.tsx:8]: This was just to fix compilation. In notifier's package.json it names the export explicitly "exported.js", so it didn't work to just import "exported"

File: wallet/src/components/SmartWalletConnection.tsx (hunk 2 above)
Comment (samsiegart) [reply wallet/src/components/SmartWalletConnection.tsx]: Good catch, I think a remnant from when I was `yarn link`ing

File: wallet/config-overrides/index.js (hunk 1 above)
Comment (samsiegart) [reply wallet/config-overrides/index.js:56]: Same

File: wallet/src/index.tsx (hunk 3 above)
Comment (turadg) [reply wallet/src/index.tsx:8]: yeah. I'm just thinking the others will need the same when they get to ESM and we should get into the habit of ESM-style imports
